CONTRIBUTING.md

```diff
@@ -244,7 +244,21 @@ The following example is for releasing a patch for `v0.69.0` -> `v0.69.1`.
 - Create PRs with base set to that release branch
   - When the PR is merged, a changeset PR is created
   - When the changeset PR is merged into the release branch, the next patch version is released and the commit is tagged (e.g. `v0.69.1`)
-- After release, delete the release branch from GitHub
+- After release, the release branch will be automatically deleted
+
+# Patching latest release
+
+Imagine the scenario:
+
+1. We release `v0.80.0`
+1. One day later, we have a new changesets PR that will bump things to `v0.81.0`
+1. Before releasing `v0.81.0`, we find an issue and need to make a `v0.80.1` patch
```

Comment:
Would it work if the found issue needed a breaking change (minor bump) to be fixed?

Reply:
Currently not because of [this check](https://github.com/FuelLabs/fuels-ts/blob/e165e37307d65deda99f675723ea26470bf807d8/.github/workflows/pr-validate-changesets.yaml#L129). I can change the logic so that releases older than `latest` can only be patched, but `latest` can have a minor bump. What do you think?

Reply:
That's alright.

Reply:
This has been implemented [here](https://github.com/FuelLabs/fuels-ts/pull/2497/files#diff-900c14db30cf12f70161bd32e69363c032130443d4f052508d970d7d2fbfa472R124).